README.md

```diff
@@ -1 +1,149 @@
+
+Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+ESlint (is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs).
+🛠 Built With Tech Stack Client Server Database
+
+# 🚀 Live Demo
+Please use the following link to access the website:
+[live demo](https://riyana5678.netlify.app/)
+[live presentation](https://www.loom.com/share/fddfc6960af443308443f38ba92144b6)
+
+💻 Getting Started
+To get a local copy up and running, follow these steps.
+
+# Prerequisites
+Have a local version control like Git. Which is a open source distributed version control system designed for source code management
+A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+Download waves to check project accessibilty
+
+
+# Setup
+Clone the repo by running the command
+
+ git clone https://github.com/Ibtisam34/Metrics-webapp.git
+
+# Install
+Install this project with:
+
+Open the directory of the project
+
+cd Space-Travelers
+
+Open the directory of the src
+
+cd src
+Open the index.html file
+
+Click and open the index.html file
+
+# Usage
+
+To run the project, execute the following command:
+
+Please run the following to render on the browser
+
+npm start
+
+Run tests
+
+To run tests, run the following command:
+
+Please run the following to test
+npm run test
+
+# Deployment
+You can deploy this project using:
+
+Please run the following to deploy
+npm run deploy
+
+
+
+👥 Authors
+Mention all of the collaborators of this project.
+
+👤 **Riyana**
+
+- GitHub: [@ibtisam34](https://github.com/ibtisam34)
+- Twitter: [@Queenjin2](https://twitter.com/Queenjin2)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/falis-abdikani)
+
+🔭 Future Features
+   continue modifying till I get the best app from my app
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+(back to top)
+
+⭐️ Show your support
+Write a message to encourage readers to support your project
+
+If you like this project...
+
+🙏 Acknowledgments
+Microverse for providing the Design and guideline for this project.
+(back to top)
+
+❓ FAQ
+How can I find the node modules folder?
+
+The node modules folder was added to the gitignore file, so you need to install it manually
+Please run the following to install node modules
+
+  npm install
+
+Which index.html file should I open to make some changes in src folder or dist folder?
+
+Please make change only to Html, CSS and JS files in the src directory
+(back to top)
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
[OPTIONAL] Some of. your subheadings are squeezed together like authors, future features, contribution, show your support, license and faq. Make them bold also for example:
```
## Faq
```
The two `#` and a space will make it a subheading or a single `#` for a main heading